app/models/menu.rb

```diff
@@ -1,5 +1,5 @@
 class Menu < ApplicationRecord
   validates_presence_of :name
-
+  belongs_to :restaurant
```

Comment:
Does this solve the task about creating a relationship between restaurant and menu models? :) 
We had it as a task in an upcoming feature: https://www.pivotaltracker.com/story/show/128857113